lib/secret-storage/tests/test_secret_storage.c

```diff
@@ -286,6 +286,7 @@ Test(secretstorage, test_rlimit)
     {
       cr_assert_not(secret_storage_store_string(key_fmt, "value"), "offending_key: %s", key_fmt);
     }
+  g_free(key_fmt);
```

Comment:
Good catch. I am thinking maybe there is no need to have key_fmt on the heap at all, and could be allocated on the stack, removing a little complexity from the unit test.